modules/standard/IO.chpl

```diff
@@ -3886,7 +3885,7 @@ proc channel.readline(arg: [] uint(8), out numRead : int, start = arg.domain.low
   stored in the array).
 
   :arg arg: A 1D DefaultRectangular non-strided array storing int(8) or uint(8) which must have at least 1 element.
-  :arg maxSize: The maximum amount of bytes to read.
+  :arg maxSize: The maximum number of bytes to return.
```

Comment:
I don't think using the word return here is the most appropriate since the function does not return the number of bytes read. We could either say the number of bytes read into the array or number of bytes stored into the array in order to disambiguate the bytes read from the file and the bytes stored into the array.

Reply:
Oh might also want it to say that `maxSize` defaults to the size of the array.